### DISCUSSION TITLE
Dpi проводит нам по губам...

### DISCUSSION THREAD

**Discussion by ghost** [xvzc/SpoofDPI#259] [General]
То работает то не работает.

**Comment by kmskmskms** [comment]
со вчерашнего дня перестало работать

**Comment by vadimkazennikov** [comment]
аналогично, с 27 интерфейс трубы загружается, но видосы не грузятся

**Comment by Rvcube** [comment]
подтверждаю. Работает с большими тормозами

**Comment by malec985** [comment]
Вообще не работает. Грузит только интерфейс youtube; видео, увы, не грузятся

UPD. Заработало со сл. параметрами, пробуйте.
spoofdpi --enable-doh --window-size 0

**Reply by Qil4comDam3g4** [reply]
> Вообще не работает. Грузит только интерфейс youtube; видео, увы, не грузятся
> 
> UPD. Заработало со сл. параметрами, пробуйте. spoofdpi --enable-doh --window-size 0

Хз, у меня на маке не работает даже с такими параметрами...

**Comment by ghost** [comment]
У меня вообще ютуб заработал напрямую без обходов, большие поставки пришли в ркн...

**Comment by bykoff** [comment]
Бортовой журнал «неработы» спуфа – 01.10.2024 — не работает.

**Comment by Necron666759** [comment]
До сегодняшнего дня приложение работало исправно. К сожалению ютуб снова успешно блокируется.

**Reply by Gen5551** [reply]
@банный ркн!

**Comment by Gen5551** [comment]
Несколько дней назад после отключения 8 октября(ремонт у поставщика интернет провайдера) перестал вообще работать. пробовал все параметры но только страница а видео так и не прогружается.
